rpmio/rpmio.c

```diff
@@ -382,7 +382,7 @@ static ssize_t fdWrite(FDSTACK_t fps, const void * buf, size_t count)
 
 static int fdSeek(FDSTACK_t fps, off_t pos, int whence)
 {
-    return lseek(fps->fdno, pos, whence);
+    return (lseek(fps->fdno, pos, whence) == -1 ? -1 : 0);
```

Comment:
Mm, sorry for missing this earlier, was just about to merge when I spotted something strange with the parenthesis: enclosing it all in parenthesis makes no sense and doesn't help readability either, it should be:

```
return (lseek(fps->fdno, pos, whence) == -1) ? -1 : 0;
```